en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+
+WSO2 IS has introduced an external tool that re-encrypts internal data after rotation of the configured symmetric data encryption key. Here mainly we have considered the re-encryption of the Identity and registry databases and some configuration files. Apart from that the tool will sync end user data that gets generated in the live system to the new setup.
+
+The external java client that performs the above tasks can be created by following the steps [here](../../administer/blue-green-data-encryption-keyrotation/#how-to-create-the-key-rotation-tool).
+
+## Tables that support re-encryption 
+
+At the moment below tables are supported for re-encryption
+
+??? info "Tables that support re-encryption"
+
+    | **Table**                                | **Fields**                                                                      |
+    |------------------------------------------|---------------------------------------------------------------------------------|
+    | IDN_IDENTITY_USER_DATA                   | TOTP secretKey and verifiedSecretKey claims                                     |
+    | IDN_OAUTH2_AUTHORIZATION_CODE            | OAuth2 authorization codes                                                      | 
+    | IDN_OAUTH2_ACCESS_TOKEN                  | OAuth2 access and refresh tokens                                                | 
+    | IDN_OAUTH_CONSUMER_APPS                  | Consumer secrets                                                                | 
+    | WF_BPS_PROFILE                           | BPS profile password                                                            | 
+    | WF_REQUEST                               | WF request credentials                                                          |   
+    | REG_PROPERTY                             | Keystore passwords, Keystore privatekeyPass and Entitlement subscriberPasswords | 
+
+## Configuration files that support re-encryption
+
+At the moment below configuration files are supported for re-encryption
+
+??? info "Configuration files that support re-encryption"
+    
+    | **Configuration file**            | **Path**                                                          | **Property** |
+    |-----------------------------------|-------------------------------------------------------------------|--------------|
+    | Event publishers                  | `/repository/deployment/server/eventpublishers` files             | Password     | 
+    | Super tenant secondary userstores | `/repository/deployment/server/userstores/<userstore>` files      | Password     | 
+    | Tenant secondary userstores       | `/repository/tenants/<tenant_id>/userstores/<userstore>` files    | Password     |      
+    
+## Tables that support syncing
+
+At the moment below tables are supported to be synced during key rotation from the old setup to the new setup. Any other data won’t be persisted in the new setup.
+
+??? info "Tables that support syncing"
```

Comment:
Avoid the collapsible?

Reply:
Sure, addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808